includes/blocks/class-sensei-course-progress-block.php

```diff
@@ -61,7 +61,7 @@ public function render_course_progress( $attributes ): string {
 		$percentage    = $stats['completed_lessons_percentage'];
 
 		// translators: %1$d number of lessons completed, %2$d number of total lessons, %3$s percentage.
-		$attributes['label']      = sprintf( __( '%1$d of %2$d lessons completed (%3$s)', 'sensei-lms' ), $completed, $total_lessons, $percentage . '%' );
+		$attributes['label']      = sprintf( __( '%1$d of %2$d lessons completed (%3$s)', 'sensei-lms' ), $completed, $total_lessons, round( $percentage ) . '%' );
```

Comment:
Does it make more sense to use `floor` here? To me, a 99.5% completed test is not 100% completed. 😄